resources/base.scss

```diff
@@ -86,7 +86,6 @@ th {
 .form-area {
     display: inline-block;
     background: $color_primary5;
-    padding: 5px 10px 10px;
```

Comment:
Again, let's not change it globally unless you're sure this won't adversely affect any other `form-area` across the entire site.